CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+## Contributing
+Thank you for your interest in contributing to Codewind.
+
+#### Before you contribute
+- Contributions must follow the Eclipse organization specifications. Refer to the [Git commit instructions](https://www.eclipse.org/projects/handbook/#resources-commit).
+
+#### Adding contributions
+- See the [Codewind GitHub Workflows](https://wiki.eclipse.org/Codewind_GitHub_Workflows) wiki.
+
+## Raising an issue
+
+#### Before you raise an issue
+- Search the [list of issues](https://github.com/eclipse/codewind-eclipse/issues) to see if your issue has already been raised.
+- See the [Codewind documentation](https://www.eclipse.org/codewind/docindex.html) to see if existing documentation includes the issue.
+
+#### Creating a new issue
+If you do not see your issue, please [select the issue type in the Codewind repository to open a new issue](https://github.com/eclipse/codewind/issues/new/choose).
+- **Bugs:** Complete the bug template to report problems found in Codewind.
+- **Enhancements:** Complete the enhancement template to suggest improvements to Codewind.
+- **Questions:** Complete the question template to ask a question about Codewind.
+
+## Contact us
+If you have questions, please contact us:
+- [Visit us on Slack.](https://slack-invite-ibm-cloud-tech.mybluemix.net/)
```

Comment:
Is there a reason why we have the Kabanero slack link in here. From Codewind perspective, we are using mattermost.